### ISSUE TITLE
Two bugs in sites_cultivars

### ISSUE THREAD

**Issue by dlebauer** [PecanProject/bety#521]
* [x] 1. need uniqueness constraint to prevent duplicates of `site_id` ~~(site_id, cultivar_id)~~
* [x] 2. indexing should use the assigned id range for the database. currently appears to start from 1 on all databases.

**Comment by dlebauer** [comment]
To remove duplicates from sites_cultivars

```sql
DELETE FROM sites_cultivars 
WHERE id IN (SELECT id
              FROM (SELECT id,
                             ROW_NUMBER() OVER (partition BY site_id, cultivar_id ORDER BY id) AS rnum
                     FROM sites_cultivars ) t
              WHERE t.rnum > 1);
```

**Comment by gsrohde** [comment]
@dlebauer The duplicate-removal code looks fine.  The bug fixes are in pull request #522, which I've asked @robkooper to review.

**Comment by gsrohde** [comment]
Fixed by release 4.18.
